spec/requests/articles/articles_spec.rb

```diff
@@ -95,16 +95,15 @@
         expect(response.body).not_to include(organization_article.title)
       end
 
-      it "contains the full user URL" do
-        expect(response.body).to include("<link>#{URL.user(user)}</link>")
-      end
-
-      it "contains a user composite profile image tag", :aggregate_failures do
-        expect(response.body).to include("<image>")
-        expect(response.body).to include("<url>#{app_url(user.profile_image_90)}</url>")
-        expect(response.body).to include("<title>#{community_name}: #{user.name}</title>")
-        expect(response.body).to include("<link>#{URL.user(user)}</link>")
-        expect(response.body).to include("</image>")
+      it "contains a user' name, link, and composite profile image tag", :aggregate_failures do
+        expect(response.body).to include(
+          "<image>",
+          "<url>#{app_url(user.profile_image_90)}</url>",
+          "<title>#{community_name}: #{user.name}</title>",
+          "<link>#{URL.user(user)}</link>",
+          "</image>",
+          "<dc:creator>#{user.name}</dc:creator>",
```

Comment:
this is a nice cleanup. I don't think I'd seen include with an array of substrings before, TIL.

Reply:
it's one of my favorite underrated feature